scripts/github/test-workflows.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+cd ../..
+
+
+# Shows available actions
```

Comment:
I think we can remove this right? No need for us to explain how command line tools work I think. And uncommented code only creates confusion most of the time.

Reply:
Sure 👍